packages/google_maps_flutter/google_maps_flutter/example/android/app/build.gradle

```diff
@@ -66,6 +63,9 @@ android {
         api 'androidx.test:core:1.2.0'
         testImplementation 'com.google.android.gms:play-services-maps:17.0.0'
     }
+    lint {
```

Comment:
Why did you change this?

Reply:
Moved to https://github.com/flutter/packages/pull/3408